frontend/src/scenes/insights/insightLogic.ts

```diff
@@ -635,9 +596,9 @@ export const insightLogic = kea<insightLogicType>({
             }
         },
         saveInsight: async ({ setViewMode }) => {
-            const insightId =
+            const insightNumericId =
```

Comment:
`insightNumericId` is a bit ambiguous because short IDs can be numeric too. However I'm not sure what would be the ideal name, could have something to do with being serial/autoincremented/the primary key. 🤔

Reply:
I'd like to do one more pass on this after this PR. 1) replace `dashboardItemId` with `insightId` everywhere, and 2) replace `insightNumericId` with something else... perhaps `insightPrimaryKey`? This change to `insightNumericId` frees up all the other `insightId`-s to have string short-id values.